config/webpack.dev.config.js

```diff
@@ -77,7 +77,9 @@ module.exports = Merge.smart(commonConfig, {
     port: 18011,
     proxy: Object.keys(apiEndpoints).reduce(
       (map, endpoint) => {
-        map[apiEndpoints[endpoint]] = targetUrl;// eslint-disable-line no-param-reassign
+        map[apiEndpoints[endpoint]] = {
+          target: targetUrl,
+        };
```

Comment:
I've changed this reduce function to return an object of the form:

proxy: {
    "aaa": {...},
    "bbb": {...},
}

as I think it will be easier to add to, if necessary. I don't think it's necessary, but thought I'd include it to field thoughts on it.